v2/event/eventcontext_v03_test.go

```diff
@@ -162,10 +162,6 @@ func TestGetMediaTypeV03(t *testing.T) {
 		"nil": {
 			want: "",
 		},
-		"just encoding": {
```

Comment:
why delete the test?

Reply:
Because per spec https://tools.ietf.org/html/rfc7231#section-3.1.1.1, it seems to me that a content type with just encoding doesn't make sense